experimenter/tests/integration/nimbus/test_experimenter_ui.py

```diff
@@ -292,3 +292,24 @@ def test_summary_timeline_release_date(
 
     summary.wait_for_timeline_visible()
     summary.wait_for_release_date()
+
+
+@pytest.mark.nimbus_ui
+@pytest.mark.skipif(
+    any(
+        app in os.getenv("PYTEST_ARGS")
+        for app in ["FOCUS_IOS", "IOS", "FENIX", "FOCUS_ANDROID"]
+    ),
```

Comment:
You can just say this as 

```py
os.getenv("PYTEST_ARGS") not in [.....]
```

Reply:
I was thinking that we're going to want this to run for non-desktop apps, so it makes more sense to say "skip all of the mobile apps" than it does to say "skip if it's not desktop" and then have to edit it to say "skip if it's not desktop or relay or monitor"

Reply:
No I mean, we only want to run this for the mobile apps, so skip if app not in [fenix, ios, focusa, focusi].  The list is correct I just mean you don't need to use `any(x in y for x in z)`

Reply:
I know I just mean, you don't need to use `any(x in y for x in z)` formulation.  You're only comparing a single value so you just have to write `x in [.....]`, so if you want to skip the mobile apps or run only for the mobile apps, you can just use `in` or `not in`.